### ISSUE TITLE
Segfault when a process is attached without specifying exe

### ISSUE THREAD

**Issue by cuviper** [dyninst/dyninst#146]
It has long worked fine to attach to existing processes without specifying their executable path.  However, this has recently led to a segfault in `linux_process::computeAddrWidth()`, when it tries to open the exe to determine endianness.  With that fixed, there's a similar problem in `Codegen::generate()`.  Both of these were changed in c8d74813e605b675f28f0adb19d5b8a0cb59db81.

This is a regression from 9.1.  I have fixes to send shortly.

**Comment by mcfadden8** [comment]
Ugh.  This one was introduced by me..  It would have helped if I read the comment above the code which said:

```C
    /**
     * It's surprisingly difficult to figure out the word size of a process
     * without looking at the files it loads (we want to avoid disk accesses).
```

Glad to hear that a fix is already in mind and thanks for cleaning up my mess..

**Comment by cuviper** [comment]
It might not be in the testsuite.  I'll take a look at that too.

**Comment by mcfadden8** [comment]
Which branch should I clone from to test this?

**Comment by cuviper** [comment]
It's not clear to me how to best test this, so I filed dyninst/testsuite#13.

**Comment by cuviper** [comment]
Fixed by #147
